#point2d

float64 x
float64 y
